### ISSUE TITLE
Fields being inadvertently removed when inserting on Go 1.14

### ISSUE THREAD

**Issue by Draiken** [viant/bgc#5]
Hello!

We have an app using your library and we insert a bunch of data through `PersistSingle` and `PersistAll`.
However the same code that ran just fine on go 1.12 when I upgraded to go 1.14 started throwing errors on inserts related to missing fields.

The first one was a boolean flag which ran like this:
```
record := &migration{Version: 1, Dirty: false}
_, _, err := conn.PersistSingle(record, migrationsTable, nil)
```
Resulted in errors like these:
```
  "status": {
    "errorResult": {
      "message": "Error while reading data, error message: JSON table encountered too many errors, giving up. Rows: 1; errors: 1. Please look into the errors[] collection for more details.",
      "reason": "invalid"
    },
    "errors": [
      {
        "message": "Error while reading data, error message: JSON table encountered too many errors, giving up. Rows: 1; errors: 1. Please look into the errors[] collection for more details.",
        "reason": "invalid"
      },
      {
        "message": "Error while reading data, error message: JSON processing encountered too many errors, giving up. Rows: 1; errors: 1; max bad: 0; error percent: 0",
        "reason": "invalid"
      },
      {
        "message": "Error while reading data, error message: JSON parsing error in row starting at position 0: Missing required field: dirty.",
        "reason": "invalid"
      }
    ],
    "state": "DONE"
  },
```
I am guessing something changed on golang that made [normalizeValue](https://github.com/viant/bgc/blob/master/insert.go#L66-L107) remove fields when it should not.

Versions:
```
bgc 0.7.0
dsc 0.10.2
```
I should note that I tried upgrading both `dsc` and `bgc` to the latest versions, and the errors persisted.

**Comment by adranwit** [comment]
looking into it,

**Comment by adranwit** [comment]
It looks like in 1.14 the following function returns true for bool field with false value
Initially, this function was used to detect time.Time or struct value which has not been initialized, 
will work to  apply patch to viant/dsc

//IsZero returns true if input is a zeroable
func IsZero(input interface{}) bool {
	if zeroable, ok := input.(Zeroable); ok {
		return zeroable.IsZero()
	}
	return false
}

Detail to the patch
https://github.com/viant/dsc/commit/f8e07051c086ce5a936d65525f9baad257fcfb75

**Comment by adranwit** [comment]
Issue resolved with viant/dsc v0.16.2

**Comment by Draiken** [comment]
Thank you very much!
